docroot/legalcode/by-nc-nd_4.0_el.html

```diff
@@ -706,52 +706,6 @@ <h6><a href="https://creativecommons.org/about/contact">We'd love to hear from y
     </div>
   </footer>
 </div>
-<!-- Site Footer Start - DO NOT DELETE -->
```

Comment:
@aldenstpage this says `DO NOT DELETE`, is it okay to remove it?

Reply:
I see other files with the same issue too.

Reply:
They're temporary anchors that get replaced by the scripts. The "DO NOT DELETE" section is for the translators working on the templates. If you check out this PR and open the files in question, you'll see that they render correctly with the footer intact.